Tests/InfrastructureLayer/Settings/SettingsTest.cpp

```diff
@@ -51,7 +51,12 @@ TEST_F(SettingsTest, packetTitle)
     EXPECT_EQ(settings_->packetTitle(), "Gen5");
 }
 
-TEST_F(SettingsTest, exchangeName)
+TEST_F(SettingsTest, logLevel)
 {
-    EXPECT_EQ(settings_->exchangeName(), "hermesExchange");
```

Comment:
Do we not need this test anymore?

Reply:
Hm you're right, must've accidentely deleted it